doc/material-controls-extensions.md

```diff
@@ -0,0 +1,38 @@
+# Material Control Extensions
+
+## Icon
+This feature allows to put an icon on some supported control. Those icons could be any kind of _Icon_: `<BitmapIcon />`, `<FontIcon />`, `<SymbolIcon/>`, or `<PathIcon />`.

```

Comment:
```suggestion
This feature allows for the addition of icon on the supported controls. Those icons could be any of the [`IconElement`](https://docs.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.iconelement)s: `<BitmapIcon />`, `<FontIcon />`, `<PathIcon />` or `<SymbolIcon />`.
```

Reply:
might want to clarify which icons are supported and on what platforms here
i have a feeling that we probably dont support all of them..?
perhaps we can expand the [IconElement doc](https://platform.uno/docs/articles/implemented/windows-ui-xaml-controls-iconelement.html) and link it here? or, it is already somewhere?